packages/labs/src/components/select/select.tsx

```diff
@@ -248,8 +248,9 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
         Utils.safeInvoke(popoverProps.popoverWillClose);
     }
 
-    private handleQueryChange = (event: React.FormEvent<HTMLInputElement>) => {
-        this.setState({ query: event.currentTarget.value });
+    private handleQueryChange = (e: React.FormEvent<HTMLInputElement>) => {
```

Comment:
Ha, was just trying to stay consistent with previous feedback from @adidahiya: https://github.com/palantir/blueprint/pull/1275#discussion_r126009905
What do you think? Happy to change back to `event` across `labs`, let me know.